src/components/LoginComponent.jsx

```diff
@@ -148,101 +124,60 @@ class LoginComponent extends Component {
   render () {
     let { classes, renderReceiveLogin, renderReceiptLogin, isMainNet } = this.props
 
-    const envSuffix = isMainNet ? 'MainNet' : 'TestNet'
-
     if (renderReceiveLogin) {
       return this.renderReceiveLogin()
     } else if (renderReceiptLogin) {
       return this.renderReceiptLogin()
     }
 
     return (
-      <Grid container className={classNames(classes.container, classes[`container${envSuffix}`])}>
-        <Grid className={classes.faqContainer} container md={6} justify='center'>
-          <Box mt={6} mx='auto'>
-            <Grid>
+      <Grid container className={classNames(classes.container, classes.containerTestNet)}>
```

Comment:
We are not differentiate testnet and mainnet anymore, rename "containerTestNet" to something else